packages/appium/docs/en/guides/security.md

```diff
@@ -2,4 +2,61 @@
 title: Appium Server Security
 ---
 
-TODO
+The Appium team makes every effort to ensure the security of the Appium server. This is especially
+important when Appium is run in a multitenant environment, or when multiple users are running
+sessions on the same Appium server. In general, if you're running your own Appium server locally,
+and not sharing it with anyone else, and don't expose Appium's port to the wider internet, you
+should have nothing to worry about, and can safely enable all Appium's features.
+
+But because many Appium users might not be able to guarantee such a safe environment, the Appium
+team puts many features behind a security protection mechanism which forces system admins (the
+people that are in charge of starting the Appium server) to opt-in to these features explicitly.
+(Third-party driver and plugin authors can also [hide behaviour behind security
+flags](../ecosystem/build-drivers.md).)
+
+For security reasons, Appium client sessions can _not_ request feature enablement via capabilities.
+This is the responsibility of the server admin who launches the Appium server.
+
+## Security Server Args
+
+The [Server CLI Args](../cli/args.md) doc outlines three relevant arguments which may be passed to
+Appium when starting it from the command line:
+
+|Parameter|Description|
```

Comment:
oof, run prettier 😄

Reply:
![Screen Shot 2022-11-02 at 10 26 51](https://user-images.githubusercontent.com/605053/199559541-113994fb-8b01-4038-adf6-d427126a7263.png)
running a table formatter makes it look so much more shitty in my editor, i usually don't bother